pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -80,21 +69,27 @@ def emit_text(self, content: str) -> None:
     def emit_newline(self) -> None:
         self.output.append("")
 
-    def emit_table_separator(self) -> None:
-        self.output.append(table_separator)
-
     def emit_author_table_separator(self, max_name_len: int) -> None:
         author_table_separator = "=" * max_name_len + "  " + "=" * len("email address")
         self.output.append(author_table_separator)
 
-    def emit_pep_row(self, pep_details: dict[str, int | str]) -> None:
-        self.emit_text(column_format(**pep_details))
+    def emit_pep_row(self, *, type: str, status: str, number: int, title: str, authors: str) -> None:
+        self.emit_text(f"   * - {type}{status}")
+        self.emit_text(f"     - :pep:`{number} <{number}>`")
+        self.emit_text(f"     - :pep:`{title.replace('`', '')} <{number}>`")
+        self.emit_text(f"     - {authors}")
 
     def emit_column_headers(self) -> None:
         """Output the column headers for the PEP indices."""
-        self.emit_table_separator()
-        self.emit_pep_row({"status": ".", "type": ".", "number": "PEP", "title": "PEP Title", "authors": "PEP Author(s)"})
-        self.emit_table_separator()
+        self.emit_text(".. list-table::")
+        self.emit_text("   :header-rows: 1")
+        self.emit_text("   :widths: auto")
+        self.emit_text("   :class: pep-zero-table")
+        self.emit_newline()
+        self.emit_text("   * - ")
+        self.emit_text("     - PEP")
+        self.emit_text("     - PEP Title")
+        self.emit_text("     - PEP Author(s)")
```

Comment:
"Author(s)" is a bit ugly, for a table heading can we just put "Authors" or "Author"?

Also I think it's clear that the title is the _PEP_ title, and the authors are the _PEP_ authors, in a list of PEPs on the PEP index page on https://peps.python.org/ :)

```suggestion
        self.emit_text("     - PEP")
        self.emit_text("     - Title")
        self.emit_text("     - Authors")
```